user-org-actor/src/main/java/org/sunbird/service/UserServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.sunbird.service;
+
+import org.sunbird.dao.IUserDao;
+import org.sunbird.dao.UserDaoImpl;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+
+import java.util.Map;
+
+/**
+ * @author Amit Kumar
+ * This class will contains user entity related CRUD method
+ */
+public class UserServiceImpl implements IUserService{
+
+    private IUserDao userDao;
+
+    public UserServiceImpl() {
+        userDao = new UserDaoImpl();
+    }
+
+    @Override
+    public Response createUser(Request request) {
+        Response response = new Response();
+        try {
+            io.opensaber.pojos.Response osResponse = userDao.addUser(request.getRequest());
```

Comment:
Perhaps we can unify this Response. Maybe then we don't need the service layer...The Akka controller is the service layer. I see no benefit in keeping an internal service/dao interfaces. Lets discuss.

Reply:
need discussion.

Reply:
fixed